docroot/sites/ighn.international.uiowa.edu/modules/ighn_core/sass/ighn.scss

```diff
@@ -1,6 +1,73 @@
-@import "../../../../../themes/custom/uids_base/uids/assets/scss/_variables.scss";
-@import "../../../../../themes/custom/uids_base/uids/assets/scss/_utilities.scss";
-
-.field--name-field-ighn-person-credentials {
-  @include card-subtitle;
-}
```

Comment:
We should be extending UIDS here to keep things on-brand.

Reply:
@richardbporter Setting the ighn_person fields to the uids card fields in the template applies the uids styling... is that not enough?

Reply:
Maybe @joewhitsitt can clarify things here but I thought the point of this Yarn workspace was to do the imports and make minor style adjustments, keeping the brand in mind. If no style adjustments are necessary, is the workspace overkill? Is attaching the library enough?

Reply:
@dnkemper the person teaser/card templates provides general styling, but the yarn workspace/library attach was setup this way so that you can utilize variables and utilities to extend uids to cover ighn unique fields with styles from uids_base.

ighn person isn't too big of a departure from the person content type (just more fields) and to follow the brand would align with the styles already done for the person content type and the paragraphs views lists. docroot/themes/custom/uids_base/scss/content/node--type--person.scss, docroot/themes/custom/uids_base/scss/paragraphs/uiowa-paragraphs-lists.scss

For example this is what an "on-brand" person card looks like:
<img width="427" alt="Screen Shot 2020-06-26 at 11 39 51 AM" src="https://user-images.githubusercontent.com/4663676/85889727-a2fff800-b7b1-11ea-86f0-d6580565d653.png">

_https://hawkeyemarchingband.local.drupal.uiowa.edu/people-0, switch to uids_base and change the featured content paragraph to grid style._

Reply:
@joewhitsitt I would understand/only extend uids_base if this site wasn't already launched with a design... but I didn't realize I wasn't able to use part of the theme I had already billed them for.

Reply:
@joewhitsitt @richardbporter she also made specific requests that don't align with the uids styling... like sizing/line height etc